### DISCUSSION TITLE
Custom style for Ant Design Collapse header

### DISCUSSION THREAD

**Discussion by thxnhi** [ant-design/ant-design#46483] [Q&A]
I want to create a custome style for Ant Design Collapse header. I want to change its default align-items to center.
![c79daf9a-eceb-41cf-9538-9e3cdc09b4a8](https://github.com/ant-design/ant-design/assets/84732935/48ab6a2f-33a1-4715-a11f-226f3f362f6f)

**Comment by panagopoulos** [comment]
On < Collapse > set a className (e.g. className = 'collapse-classname'). Then on CSS use 
```
.collapse-classname > :nth-child(n) > :first-child {
  display: 'flex';
  align-items: 'center;
}
```
